templates/feeds-index.item

```diff
@@ -1,10 +1,12 @@
       <div class="row mb-4">
```

Comment:
Really `mb-4`?

Reply:
this is just a margin between rows. Should we remove it?

Reply:
![image](https://github.com/achernyakevich/smotrim-rss/assets/122963199/3439c5bd-a707-4b18-9d85-d04d463b2f5a)